app/templates/auth/change-password.html

```diff
@@ -0,0 +1,122 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Change password – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": dashboard_url,
+        "label": "Your account"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+{% if form.errors|length > 1 %}
```

Comment:
OK - this pattern is used across the app as a 'there's multiple things wrong with this form' message, with bookmark links to jump to that error in the form. 

A single error would still be shown at field level, but it feels a bit like overkill to have two messages for one error? I can check with a designer in case this pattern usage has changed?

Reply:
I think this is a thing we bumped into when doing the supplier details stuff in the last mission. The word from Laurence was that for consistency that even on a single question page we had the validation masthead as well as the field level thingy.